client/state/action-types.js

```diff
@@ -75,6 +75,11 @@ export const AUTOMATED_TRANSFER_STATUS_SET = 'AUTOMATED_TRANSFER_STATUS_SET';
 export const BILLING_RECEIPT_EMAIL_SEND = 'BILLING_RECEIPT_EMAIL_SEND';
 export const BILLING_RECEIPT_EMAIL_SEND_FAILURE = 'BILLING_RECEIPT_EMAIL_SEND_FAILURE';
 export const BILLING_RECEIPT_EMAIL_SEND_SUCCESS = 'BILLING_RECEIPT_EMAIL_SEND_SUCCESS';
+export const BILLING_TRANSACTION_CLEAR_ERROR = 'BILLING_TRANSACTION_CLEAR_ERROR';
```

Comment:
Nit: usually we keep the action/verb at the end of the action name, so I'd call this one `BILLING_TRANSACTION_ERROR_CLEAR` instead of `BILLING_TRANSACTION_CLEAR_ERROR`